manifests/instance.pp

```diff
@@ -286,6 +286,7 @@
   Variant[Stdlib::Absolutepath, Enum['']] $unixsocket            = "/var/run/redis/redis-server-${name}.sock",
   Stdlib::Absolutepath $workdir                                  = "${redis::workdir}/redis-server-${name}",
 ) {
+  include systemd::systemctl::daemon_reload
```

Comment:
Perhaps this should be inside the if condition below?

Reply:
Sure, will do.